<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordEditView</class>
 <widget class="QWidget" name="RecordEditView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>新建就诊记录</string>
   </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPatient">
       <property name="text">
        <string>患者：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboPatient"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDoctor">
       <property name="text">
        <string>医生：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboDoctor"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelVisitDate">
       <property name="text">
        <string>就诊日期：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateVisit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelVisitType">
       <property name="text">
        <string>就诊类型：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboVisitType">
       <item>
        <property name="text">
         <string>门诊</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>急诊</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>复诊</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelDiagnosis">
       <property name="text">
        <string>诊断：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="editDiagnosis"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelSymptoms">
       <property name="text">
        <string>症状：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="editSymptoms"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
